doc/source/main-documentation/qibolab.rst

```diff
@@ -509,10 +494,13 @@ For example:
 
 .. testcode:: python
 
+    from qibolab.pulses import PulseSequence, Delay
```

Comment:
No, but this still happens, see for example a few lines above in line 461 of the same file. Maybe sphinx keeps the imports from previous cells and I only needed to import `Delay` here, I just added both because I noticed that `PulseSequence` was also missing. However, `Delay` is also imported in a previous cell, so I am not sure if that theory is correct.

Reply:
Maybe we should take the time of investigating this, even if probably not immediately. It could be confusing for users (as it was in #801), and maybe there is a configuration to change this behavior.

Reply:
#828. I wouldn't go into that here given that this PR is already big. We could also do that investigation for 0.1.